src/main/webapp/resources/js/pages/analysis/components/AnalysisSamples.jsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file get the sample data from the server, and loads
+ * the Samples and Reference File components
+ */
+
+/*
+ * The following import statements makes available all the elements
+ * required by the component
+ */
+import React, { useContext, useEffect, useState } from "react";
```

Comment:
`useState` is never used.

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6